processing/src/main/java/io/druid/segment/QueryableIndexStorageAdapter.java

```diff
@@ -172,6 +185,9 @@ public Capabilities getCapabilities()
   @Override
   public ColumnCapabilities getColumnCapabilities(String column)
   {
+    if (index.getColumn(column) == null) {
```

Comment:
would be nice to save `index.getColumn(column)` so we don't have to do the lookup twice.